db/migrate/20200423034154_create_package_sets.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePackageSets < ActiveRecord::Migration
+  def change
+    create_table :package_sets do |t|
+      t.integer :package_type_id
+      t.text :description
+
+      t.timestamps null: false
+    end
+
+    add_column :packages, :package_set_id, :integer
+    remove_column :packages, :set_item_id, :integer
```

Comment:
Is 'set_item_id' column required for migration or just using item relationship?

Reply:
Yes, using the normal item relationship. Not needed